demos/next-i18next/next.config.js

```diff
@@ -0,0 +1,14 @@
+const { i18n } = require('./next-i18next.config')
+
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
+  eslint: {
+    // Warning: This allows production builds to successfully complete even if
+    // your project has ESLint errors.
+    ignoreDuringBuilds: true,
```

Comment:
More of a general question as I'm noticing that this exists within `demos/middleware` (and now here) but not in the other demo projects:

Is this a configuration setting that we're ok with having in our `next.config.js` files for demo projects in general? Or is there are particular eslint error that we're trying to avoid addressing within this project that is time consuming to address that we're ok with ignoring because it's a demo project?